docs/faq.md

```diff
@@ -1 +1,50 @@
-TBA.
\ No newline at end of file
+# Frequently Asked Questions
+
+## What are Fleet-owned Resources on the Hub and Member Clusters, and Can These Fleet-owned Resources be Modified by the User?
+
+- **Namespaces:**
+    - `fleet-system` (Hub Cluster)
+    - `fleet-member-{clusterName}` (Hub Cluster)
+- **Custom Resource Definitions**
+- **Custom Resources:**
+    - `MemberCluster` (Hub Cluster, user-facing resource)
+    - `InternalMemberCluster` (Hub Cluster, internal resource)
```

Comment:
Should we use the CRD link here? you just need to point out which are user-facing resource and others are owned by fleet.

https://github.com/Azure/fleet/tree/main/charts/hub-agent/crdbases
https://github.com/Azure/fleet/tree/main/charts/member-agent/crdbases

same to fleet-networking